Bottom: 72201536adf1651e111a60e6cb0ce96f44a7fbc6
Top:    dc3142848b42cc2986aa6a4fd155122adc7f0065
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 02:53:19 +0100

Refresh of remove-old-draw-coordinate-con

---

diff --git a/src/draw.c b/src/draw.c
index 76c9a89..865bab2 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1001,8 +1001,6 @@ DrawPinOrViaNameLowLevel (PinTypePtr Ptr)
       AddPart (&box);
       return;
     }
-/*printf("DrawPin(%d,%d): x=%d y=%d w=%d h=%d\n",
-  TO_DRAW_X(Ptr->X), TO_DRAW_Y(Ptr->Y), box.X1, box.Y1, width, height);*/
 
   gui->set_color (Output.fgGC, PCB->PinNameColor);
 
diff --git a/src/line.c b/src/line.c
index d1faaec..0411afc 100644
--- a/src/line.c
+++ b/src/line.c
@@ -31,8 +31,6 @@
 #include "config.h"
 #endif
 
-#define NUDGE (TO_PCB (6))
-
 #include <math.h>
 #include <setjmp.h>
 #include <stdlib.h>
diff --git a/src/macro.h b/src/macro.h
index ddb1fb3..1a82f69 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -40,28 +40,13 @@
 #define	SWAP_IDENT			Settings.ShowSolderSide
 #endif
 
-#ifndef XORIG
-#define XORIG Xorig
-#define YORIG Yorig
-#endif
-
 #define	SWAP_SIGN_X(x)		(x)
 #define	SWAP_SIGN_Y(y)		(-(y))
 #define	SWAP_ANGLE(a)		(-(a))
 #define	SWAP_DELTA(d)		(-(d))
 #define	SWAP_X(x)		(SWAP_SIGN_X(x))
 #define	SWAP_Y(y)		(PCB->MaxHeight +SWAP_SIGN_Y(y))
-#define SATURATE(x)             ((x) > 32767 ? 32767 : ((x) < -32767 ? -32767 : (x)))
-
-#define	TO_DRAW_X(x)		TO_SCREEN((SWAP_IDENT ? SWAP_X(x) : (x)) - XORIG)
-#define	TO_DRAWABS_X(x)		(TO_SCREEN((x) - XORIG))
-#define	TO_DRAW_Y(y)		TO_SCREEN((SWAP_IDENT ? SWAP_Y(y) : (y)) - YORIG)
-#define	TO_DRAWABS_Y(y)		(TO_SCREEN((y) - YORIG))
-#define TO_LIMIT_X(x)           ((Position)(SATURATE(Local_Zoom * ((x) - XORIG))))
-#define TO_LIMIT_Y(y)           ((Position)(SATURATE(Local_Zoom * \
-                                ((SWAP_IDENT ? SWAP_Y(y) : (y)) - YORIG))))
-#define	TO_SCREEN_ANGLE(a)	(SWAP_IDENT ? SWAP_ANGLE((a)) : (a))
-#define	TO_SCREEN_DELTA(d)	(SWAP_IDENT ? SWAP_DELTA((d)) : (d))
+
 #define	TO_SCREEN_SIGN_X(x)	(SWAP_IDENT ? SWAP_SIGN_X(x) : (x))
 #define	TO_SCREEN_SIGN_Y(y)	(SWAP_IDENT ? SWAP_SIGN_Y(y) : (y))
 
@@ -71,9 +56,6 @@
 #define	TO_PCB_X(x)		TO_PCB(x) + Xorig
 #define	TO_PCB_Y(y)		(SWAP_IDENT ? \
 				PCB->MaxHeight - TO_PCB(y) - Yorig : TO_PCB(y) + Yorig)
-#ifdef __GNUC__
-#pragma GCC poison TO_SCREEN TO_SCREEN_X TO_SCREEN_Y
-#endif
 
 /* ---------------------------------------------------------------------------
  * misc macros, some might already be defined by <limits.h>
